app/(pages)/content-collection/_component/CollectionCard/MediaCard.tsx

```diff
@@ -0,0 +1,48 @@
+'use client';
+import styles from './CollectionCard.module.scss';
+import Image from 'next/image';
+
+export default function MediaCardPlaceholder() {
+  return (
+    <div className={styles.media_container}>
+      <div className={styles.edit_container}>
+        <div className={styles.title_container}>
+          <div className={styles.title}>
+            <p> Wedding name 1 </p>
+          </div>
+          <Image
+            src="/index/select-icon.png"
```

Comment:
same thing as above. Try to avoid having things that are expected to be swapped out as static text. You expect some props coming into MediaCardPlaceholder, so you should just make the component with the props, and then pass in mock data from above.